libzhlgen/test/functions/win32/1.6.9/OptionsScreen.zhl

```diff
@@ -4,3 +4,8 @@ __thiscall void OptionsScreen::OnInit(OptionsScreen *this);
 __thiscall void OptionsScreen::OnLoop(OptionsScreen *this);
 "57660f76c08d7c240883e4f0ff77fc5589e557565381ec9c0400008b07898d70fbffff0f1181f8090000c681cc02000000":
 __thiscall void OptionsScreen::Open(OptionsScreen *this, bool mainMenu);
+"57????????83e4f0ff77fc5589e557565389ce8d4d9081ecac000000":
+__thiscall void LanguageChooser::OnRender(LanguageChooser *this);
+
+"!83ec1ca1(???????\?)c1e30301d8":
```

Comment:
Looks like this might still be part of `LanguageChooser` not sure if it's part of `OptionsScreen` as it wasn't organized that way like the other globals were when I searched in Ghidra but I'm not 100% certain.


Anyways
Linux 64: (has to use offsetVariable because of RIP-relative addressing in this match) 
This signature is technically non-unique, but both matches are for languageList anyways :smile:
```
"!4889ee480335(???????\?)":
offsetVariable std::vector<std::String>* Global_OptionsScreen_languageList;
```

Linux 32: (direct match, use same declaration as Win32)
`!8d348500000000a1(???????\?)`


^^^ Both of these matches are in `LanguageChooser::OnLoop` which in Linux 32 & 64 is *before* `LanguageChooser::OnRender`
If you wanted ideal optimization, these could be declared before `LanguageChooser::OnRender` and you could use `.` at the start of the `OnRender` methods so that you only have to scan ahead from the point of this variable match and not from the start of the entire file.